pkgs/top-level/all-packages.nix

```diff
@@ -9163,7 +9168,7 @@ let
   uclibc = callPackage ../os-specific/linux/uclibc { };
 
   uclibcCross = lowPrio (callPackage ../os-specific/linux/uclibc {
-    inherit fetchurl stdenv libiconv;
+    inherit fetchurl stdenv libiconvReal;
```

Comment:
I'd prefer `libiconv = libiconvReal;` here, and leave the default.nix unchanged.